TestProjects/Eventhooks/Eventhooks.iOS/Eventhooks.iOS.csproj

```diff
@@ -86,10 +86,10 @@
   </ItemGroup>
   <ItemGroup>
     <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Contents.json">
-      <InProject>false</InProject>
+      <Visible>false</Visible>
     </ImageAsset>
     <ImageAsset Include="Assets.xcassets\Contents.json">
-      <InProject>false</InProject>
+      <Visible>false</Visible>
```

Comment:
This change shouldn't be part of this PR as it affects iOS projects

Reply:
Well, you are right, however each time I open solution in Visual Studio 15.5.2 it is changed again - I could revert it but I think newest Visual Studio wants Visible tag instead InProject. I could also revert it and prepare new PR, but it's quite a bit of overkill - what do you think?

Reply:
See https://github.com/MvvmCross/MvvmCross/pull/2493